examples/meta/src/multiclass/multiclassmachine.sg

```diff
@@ -0,0 +1,33 @@
+File fm_train_real = csv_file("../../data/fm_train_real.dat")
+File fm_test_real = csv_file("../../data/fm_test_real.dat")
+File label_train_multiclass = csv_file("../../data/label_train_multiclass.dat")
+
+#![create_features]
+Features feats_train = features(fm_train_real)
+Features feats_test = features(fm_test_real)
+Labels labels_train = labels(label_train_multiclass)
+#![create_features]
+
+#![create_kernel]
+Kernel k = kernel("GaussianKernel", log_width=2.1)
+#![create_kernel]
+
+#![create_machine]
+Machine classifier = machine("LibSVM",epsilon=0.00005)
+#![create_machine]
+
+#![create_strategy]
+MulticlassStrategy multiclass_one_vs_rest_strategy = multiclass_strategy("KernelMulticlassMachine")
+#![create_strategy]
+
+#![create_classifier]
+MulticlassMachine mc_classifier = multiclass_machine("LibSVMOneClass", kernel=k,classifier,labels_train)
```

Comment:
this is not correct, you cannot pass arguments after named arguments. In any case, you should pass named arguments, because in the background this gets called to different .put("argname", arg) calls.

Reply:
In this comment do you mean that the "classifier" variable I am making, is wrong? Should I make a variable of another name?